.github/workflows/main.yaml

```diff
@@ -34,3 +34,4 @@ jobs:
           DB_HOST: ${{ secrets.DB_HOST }}
           DB_USER: ${{ secrets.DB_USER }}
           DB_PASS: ${{ secrets.DB_PASS }}
+          ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
```

Comment:
The error is happening in the `actions/checkout@v2` step however i think this `env` block is scoped to the `Import Synapse Data` step.  i would move this env to the global environment on line 23 so that it applies to all steps in this GH action job.

Reply:
@zaro0508  It didn't like it when I added the config to `deploy.environment`:
![Screenshot 2024-10-28 at 11 44 09 AM](https://github.com/user-attachments/assets/98b06487-2250-45dd-b614-440b76cb40b2)

So I added `deploy.env` and put it there.